tests/component-plugin/src/main/java/org/mule/functional/api/component/StacktraceLogChecker.java

```diff
@@ -183,16 +187,16 @@ public boolean equals(Object obj) {
       if (this == obj) {
         return true;
       }
-      if (method != null && !method.equals(((MethodCall) obj).method)) {
+      if (method != null && ((MethodCall) obj).method != null && !method.equals(((MethodCall) obj).method)) {
```

Comment:
use a different method for this comparison, not equals. You may need to implement a hamcrest matcher for this